mlrun/api/api/endpoints/artifacts.py

```diff
@@ -137,14 +141,13 @@ def get_artifact(
     }
 
 
-@router.delete("/artifact/{project}/{uid}")
-def delete_artifact(
+def _delete_artifact(
```

Comment:
Maybe move this to the bottom of the file. Usually the convention is to have public methods before private methods, especially if that private method is a helper for multiple public methods